src/Microsoft.Identity.Client/Internal/Requests/RequestBase.cs

```diff
@@ -63,13 +63,19 @@ protected RequestBase(AuthenticationRequestParameters authenticationRequestParam
         {
             TokenCache = authenticationRequestParameters.TokenCache;
             
-            authenticationRequestParameters.RequestContext.Logger.Info(string.Format(CultureInfo.InvariantCulture,
+            // Log contains Pii 
+            authenticationRequestParameters.RequestContext.Logger.InfoPii(string.Format(CultureInfo.InvariantCulture,
                 "=== Token Acquisition ({4}) started:\n\tAuthority: {0}\n\tScope: {1}\n\tClientId: {2}\n\tCache Provided: {3}",
                 authenticationRequestParameters?.Authority?.CanonicalAuthority,
                 authenticationRequestParameters.Scope.AsSingleString(),
                 authenticationRequestParameters.ClientId,
                 TokenCache != null, this.GetType().Name));
 
+            // Log does not contain Pii
+            authenticationRequestParameters.RequestContext.Logger.Info(string.Format(CultureInfo.InvariantCulture,
```

Comment:
We might consider logging authority if it is whitelisted (well known public authorities, not oii) as no pii,

Reply:
We can log the trusted host. I will add a check for this.